src/Game.ts

```
@@ -280,13 +268,15 @@ export class Game implements ISerializable<SerializedGame, Game> {
 
       corporationCards = this.dealer.shuffleCards(corporationCards);
 
+      // Failsafe for exceding corporation pool
+      if (gameOptions.startingCorporations * players.length > corporationCards.length) {
```

I would probably throw an error here and made sure the frontend handles validation. But this doesn't hurt anyway.